test/units/testsuite-21.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+set -eux
+set -o pipefail
+
+# Save the end.service state before we start fuzzing, as it might get changed
+# on the fly by one of the fuzzers
+systemctl is-enabled end.service && SHUTDOWN_AT_EXIT=1 || SHUTDOWN_AT_EXIT=0
```

Comment:
It doesn't seem to work as expected in interactive mode where `is-enabled` returns `0` along with `static`
```sh
[145593.489054] testsuite-21.sh[83]: + systemctl is-enabled end.service
[  OK  ] Finished systemd-update-utmp-runlevel.service.
[145593.620193] testsuite-21.sh[84]: static
[145593.628080] testsuite-21.sh[83]: + SHUTDOWN_AT_EXIT=1
```

Having said that I think it would make sense to get this PR merged, turn it on on CentOS CI and polish it with the help of the CI.

Reply:
Ah, good point, I completely forgot we add the `end.service` to the transaction via `systemd.wants=`, hopefully fixed.